packages/crayons-core/package.json

```diff
@@ -50,7 +50,7 @@
     "code-sanity": "npm run lint",
     "commit": "git-cz",
     "dev": "npm run start",
-    "directional-to-logical": "npx postcss ./src/components/**/*.scss ./src/components/**/**/**/*.scss --config ./plugins/postcss-plugins/postcss.config.js --replace --no-map --verbose",
+    "directional-to-logical": "npx postcss ./src/components/**/*.scss ./src/function-components/*.scss ../../styles/**/*.scss ./src/components/**/**/*.scss ./src/components/**/**/**/*.scss --config ./plugins/postcss-plugins/postcss.config.js --replace --no-map --verbose",
```

Comment:
Can you just use `./src/**/**.scss` instead of using very specific paths since we need to lint all the scss files with in the src folder ?

Reply:
Fixed